index.html

```diff
@@ -0,0 +1,198 @@
+              </p>
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+            <div
+              class="block blocke my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2"
+            >
+              <div class="">
+                <img class="icon" src="icons/forum.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">Forum to Discuss</p>
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+
+            <div
+              class="block my-1 d-flex flex-md-column align-items-center px-3 py-3 me-md-2"
+            >
+              <div class="">
+                <img class="icon" src="icons/ignite.svg" alt="" />
+              </div>
+              <p class="program text-center mt-sm-3 ms-3">
+                Architecture Editors
+              </p>
+              <p class="program-description text-white">
+                Adobe Creative Cloud (Photoshop, Illustrator, XD), Sketch,
+                Figma, Affinity Designer, GIMP
+              </p>
+            </div>
+          </div>
+
+          <button class="join p-4 mt-5">Join CES Global Summit 2020</button>
+          <p class="desktop-buttone">
+            <a href="" class="desktop-button">SEE THE WHOLE PROGRAM</a>
+          </p>
+        </div>
+      </article>
+      <article class="speakers mt-5 px-4 py-5 bg-white">
+        <h2 class="main-program text-center text-black">Featured Speakers</h2>
+        <h2 class="baseline text-center text-white fw-bold"></h2>
+        <img src="" alt="" />
+        <div class="main-speakers my-5"></div>
+
+        <button
+          id="seemore"
+          class="more w-100 py-2 d-flex d-lg-none justify-content-center align-items-center"
+        >
+          More
+          <img class="arrow ms-2" src="icons/arrowdown.svg" alt="downarrow" />
+        </button>
+      </article>
+      <article
+        class="card partners mt-5 bg-dark d-sm-none d-md-block text-white"
+      >
+        <h2 class="partner text-center mt-5">Partner</h2>
+        <h2 class="baseline fw-bold"></h2>
+        <div class="companies">
+          <div class="row d-flex justify-content-center mb-3">
+            <img class="col-4 py-4" src="icons/mozilla.svg" alt="" />
+            <img class="col-4" src="icons/google.svg" alt="" />
+            <img class="col-4" src="icons/naver.svg" alt="" />
+          </div>
+          <div class="row d-flex justify-content-center pb-4">
+            <img class="col-6" src="icons/daumkakao.svg" alt="" />
+            <img class="col-6" src="icons/airbnb.svg" alt="" />
+          </div>
+        </div>
+      </article>
```

Comment:
- [ ] Please make sure that the partners in the "Partners" section appear in one row on the desktop view.

| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/227249462-cd25b630-f8f5-4217-ad40-b102aea372a5.png) | ![image](https://user-images.githubusercontent.com/62290545/227249570-392c15ea-3e8b-40ec-b6ca-5f2dd9b33da4.png) |